index.js

```diff
@@ -190,7 +188,7 @@ const getSettings = function(options) {
     json: json(),
     multiEntry: multiEntry({exports: false}),
     resolve: resolve({browser: true, mainFields: ['module', 'jsnext', 'main']}),
-    uglify: terser({output: {comments: 'some'}}),
+    uglify: terser({output: {comments: 'some'}, include: [/^.+\.min\.js$/]}),
```

Comment:
With this new terser option, we can always include the plugin for all browser builds. terser will only be used for the `.min.js` build.